packages/framework/src/Model/Cart/CartMigrationFacade.php

```diff
@@ -91,7 +91,11 @@ public function onKernelController(FilterControllerEvent $filterControllerEvent)
         $session = $filterControllerEvent->getRequest()->getSession();
 
         $previousCartIdentifier = $session->get(static::SESSION_PREVIOUS_CART_IDENTIFIER);
-        if ($previousCartIdentifier !== null && $previousCartIdentifier !== '' && $previousCartIdentifier !== $session->getId()) {
+        if (
+            $previousCartIdentifier !== null
+            && $previousCartIdentifier !== ''
+            && $previousCartIdentifier !== $session->getId()
+        ) {
```

Comment:
shouldn't be the first condition on the same line with `if (` ? 

does PSR-12 specify this?

Reply:
No it should not. PRS-12 exactly this one specifies here https://www.php-fig.org/psr/psr-12/#51-if-elseif-else

Reply:
👍